library/test/dotty/ShowTests.scala

```diff
@@ -64,6 +64,5 @@ class ShowTests {
     case class Car(model: String, manufacturer: String, year: Int)
 
     assertEquals("Car(Mustang,Ford,1967)", Car("Mustang", "Ford", 1967).show)
-    assertEquals("Map()", Map().show)
```

Comment:
I removed this. For some reason, it now causes a diverging implicit search in scalac

Reply:
T_T